levels/tests/resources/merge-4-solution.py

```diff
@@ -0,0 +1,26 @@
+
+    print(base64.b64decode(something_cool).decode())
+
+def print_another_cool_thing():
+    message = "You're cool (*/ω＼*)"
+    print(message)
+
+def print_yet_another_thing():
+    process = subprocess.Popen(['git', '--version'], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
This should probably be done with GitPython - subprocess is usually not needed

Reply:
I want to keep it to builtins only. The message itself doesn't matter, it just teaches you how to get gut version